liquidctl/cli.py

```diff
@@ -264,13 +269,28 @@ def main():
 
     if args['--debug']:
         args['--verbose'] = True
-        logging.basicConfig(level=logging.DEBUG, format='[%(levelname)s] %(name)s: %(message)s')
-        _LOGGER.debug('running %s', _gen_version())
+        format_color='%(log_color)s[%(levelname)s]%(name)s%(reset)s: %(message)s'
+        format_basic='[%(levelname)s] %(name)s: %(message)s'
+        level=logging.DEBUG
     elif args['--verbose']:
-        logging.basicConfig(level=logging.INFO, format='%(levelname)s: %(message)s')
+        format_color='%(log_color)s%(levelname)s%(reset)s: %(message)s'
+        format_basic='%(levelname)s: %(message)s'  
+        level=logging.INFO
     else:
-        logging.basicConfig(level=logging.WARNING, format='%(levelname)s: %(message)s')
+        format_color='%(log_color)s%(levelname)s%(reset)s: %(message)s'
+        format_basic='%(levelname)s: %(message)s'
+        level=logging.WARNING
         sys.tracebacklimit = 0
+   
+    if use_color_log:
+        formatter = colorlog.ColoredFormatter(format_color)
+        handler = logging.StreamHandler()
+        handler.setFormatter(formatter)
+        logging.basicConfig(level=level, handlers=[handler])
+    else:
+        logging.basicConfig(level=level, format=format_basic)  
```

Comment:
As this branch isn't necessary once we make colorlog required, the code can be simplified a bit.